### PR TITLE
Added Back tick support and Tests

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Tests have passed for backticks. We need this for something we are working on. Was wondering if you could approve this at your earliest convenience. Thanks and thanks for building the initial plugin!

**Review by sparachi — APPROVED** [review]
LGTM

File: test/test.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/// <reference path="../node_modules/@types/mocha/index.d.ts" />
```
Comment (ghost) [inline test/test.ts:1]: The purpose of these is to ensure that within any environment that the types will be viewable for things within a test scope

File: package.json [hunk 2]
```diff
@@ -34,8 +34,8 @@
   },
   "homepage": "https://github.com/filipesilva/rollup-plugin-angular-inline#readme",
   "devDependencies": {
-    "@types/mocha": "^2.2.39",
-    "@types/node": "^7.0.5",
+    "@types/mocha": "^2.2.41",
+    "@types/node": "^7.0.33",
```
Comment (ghost) [inline package.json:38]: Added these types to ensure that everything would work within a contained environment

File: src/index.ts [hunk 3]
```diff
@@ -78,7 +78,7 @@ function inlineResourcesFromString(content: string, urlResolver: UrlResolver) {
  * @return {string} The content with all templates inlined.
  */
 function inlineTemplate(content: string, urlResolver: UrlResolver) {
-  return content.replace(/templateUrl:\s*['"]([^'"]+?\.html)['"]/g, function (_m, templateUrl) {
+  return content.replace(/templateUrl:\s*[`'"]([^`'"]+?\.html)[`'"]/g, function (_m, templateUrl) {
```
Comment (ghost) [inline src/index.ts:81]: backtick support. We needed this because our coding linter will ding us if we don't use backticks for consistency.

**Review by ghost — COMMENTED** [review]
Added some comments for clarity on some of the decisions.

**Comment by filipesilva** [comment]
LGTM, I'll make a release with it. And thank you for fixing the tests as well.

**Comment by filipesilva** [comment]
Published as `rollup-plugin-angular-inline@1.0.1`.

**Pull request merged**